module/data/abstract.mjs

```diff
@@ -69,15 +69,16 @@ export default class SystemDataModel extends foundry.abstract.TypeDataModel {
 
   /**
    * @typedef {object} SystemDataModelMetadata
-   * @property {boolean} [singleton]                  Should only a single item of this type be allowed on an actor?
    * @property {typeof DataModel} [systemFlagsModel]  Model that represents flags data within the dnd5e namespace.
    */
 
   /**
    * Metadata that describes this DataModel.
    * @type {SystemDataModelMetadata}
    */
-  static metadata = {};
+  static metadata = Object.freeze({
```

Comment:
Do we know of any modules making use of `SystemDataModel`? If they add their own sub-types they might, right? This could be a breaking change in that case.

Reply:
I can skip freezing this data for now, though maybe we can do that in `3.1.0` to avoid module subtypes accidentally modifying the metadata of other types. I've left the metadata on the actual concrete data models frozen because I doubt anyone is inheriting from `BackgroundData` or `RaceData`.

Reply:
Yeah that sounds good to me